Tests/SourceKitLSPTests/CodeActionTests.swift

```diff
@@ -292,6 +292,48 @@ final class CodeActionTests: XCTestCase {
     XCTAssertTrue(codeActions.contains(expectedCodeAction))
   }
 
+  func testJSONCodableCodeActionResult() async throws {
+    let testClient = try await TestSourceKitLSPClient(capabilities: clientCapabilitiesWithCodeActionSupport())
+    let uri = DocumentURI.for(.swift)
+    let positions = testClient.openDocument(
+      """
+         1️⃣{
```

Comment:
Nitpick

```suggestion
     1️⃣{
```

Also, 2 spaces indentation would be nice here for consistency.